client/rpc-spec-v2/src/chain_head/tests.rs

```diff
@@ -2259,3 +2293,275 @@ async fn ensure_operation_limits_works() {
 			FollowEvent::OperationCallDone(done) if done.operation_id == operation_id && done.output == "0x0000000000000000"
 	);
 }
+
+#[tokio::test]
+async fn check_continue_operation() {
+	let child_info = ChildInfo::new_default(CHILD_STORAGE_KEY);
+	let builder = TestClientBuilder::new().add_extra_child_storage(
+		&child_info,
+		KEY.to_vec(),
+		CHILD_VALUE.to_vec(),
+	);
+	let backend = builder.backend();
+	let mut client = Arc::new(builder.build());
+
+	// Configure the chainHead with maximum 1 item before asking for pagination.
+	let api = ChainHead::new(
+		client.clone(),
+		backend,
+		Arc::new(TaskExecutor::default()),
```

Comment:
sorry @lexnv, you have to use the `tokio runtime` in the tests now since you are using tokio channels.

You to do something similar to this: https://github.com/paritytech/substrate/blob/176500ea6a68619fd409aa005bc1a89a476d802d/client/rpc/src/testing.rs#L23-#L67

Reply:
I was wrong it's only `send_timeout` that requires the tokio runtime and I bumped into it in my jsonrpsee PR